src/data/businessCase.ts

```diff
@@ -257,6 +282,6 @@ export const prepareBusinessCaseForApi = (
     alternativeBCostSavings: businessCase.alternativeB
       ? businessCase.alternativeB.costSavings
       : '',
-    lifecycleCostLines
+    lifecycleCostLines: []
```

Comment:
Why are we passing an empty array here? Why not the line items?

Reply:
Just saw your note. Let me see why that's not working... I wrote it after all.

Reply:
I'm passing an empty array right now because `lifecycleCostLines` is causing the form not be able to be saved (bad request). It's a temporary measure until we can figure out and fix why the form isn't able to be saved.

This empty array is also why I'm console logging it above to prevent ESLint errors.